components/builder-api-client/src/lib.rs

```diff
@@ -242,6 +254,12 @@ mod json_date_format {
     }
 }
 
+pub fn convert_to_json<T>(src: &T) -> Result<Json>
```

Comment:
Remove `pub`?

Reply:
Whoops, that was vestigial. I removed in https://github.com/habitat-sh/habitat/pull/7445/commits/ce4f105d78ba8b4ac483215297dde0617606fff2